sdk/tools/cabman/dfp.cxx

```diff
@@ -1133,12 +1133,13 @@ ULONG CDFParser::PerformFileCopy()
     switch (Status)
     {
         case CAB_STATUS_SUCCESS:
-            sprintf(InfLine, "%s=%s", GetFileName(SrcName).c_str(), DstName);
+            snprintf(InfLine, sizeof(InfLine)/sizeof(InfLine[0]) - 1,
```

Comment:
Why not using
```suggestion
            snprintf(InfLine, _countof(InfLine) - 1,
```

Reply:
Because this is a host-tool and the macro isn't defined there.

Reply:
_countof() is not a macro, my naive brain believes. ARRAYSIZE is one. Have you **really** already tried using countof()?

Reply:
```
FAILED: sdk/tools/cabman/CMakeFiles/cabman.dir/dfp.cxx.o 
...
/home/runner/work/reactos/reactos/src/sdk/tools/cabman/dfp.cxx: In member function ‘ULONG CDFParser::PerformFileCopy()’:
/home/runner/work/reactos/reactos/src/sdk/tools/cabman/dfp.cxx:1136:31: error: ‘_countof’ was not declared in this scope
 1136 |             snprintf(InfLine, _countof(InfLine) - 1,
      |                               ^~~~~~~~

Reply:
I finally chose to `#define _countof()` in the cabinet.h header, along with the other unix/win32 conditional definitions.